.gitignore

```diff
@@ -1,2 +1,3 @@
 **/.DS_Store
 .vscode/
+testdata
```

Comment:
`testdata` shouldn't be ignored.

Reply:
ok, removed ... but why?

Reply:
Well, you are not using `testdata` anywhere in this PR, so I don't see the need to add it to `.gitignore`. Also, `testdata` is commonly used to store test fixtures that are used when running tests, so you don't want to ignore them.

Reply:
Oh, `testdata` was an artifact of the fuzzing test for the parsing function, which is gone now.